src/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] It would be better if you write your actions following  the [ducks pattern](https://github.com/erikras/ducks-modular-redux) so I encourage you to do that